queries.sql

```diff
@@ -160,24 +160,24 @@ FULL JOIN specializations SP ON SP.vet_id = v.id
 WHERE SP.vet_id IS NULL
 GROUP BY V.name;
 -- second
-SELECT vet_name,
-       COUNT(*) AS num_visits
-FROM visits vi
-JOIN vets v ON v.id = vi.vet_id
-JOIN specializations sp ON sp.vet_id = v.id
-WHERE sp.vet_id IS NULL
-GROUP BY v.name;
+-- SELECT vet_name,
+--        COUNT(*) AS num_visits
+-- FROM visits vi
+-- JOIN vets v ON v.id = vi.vet_id
+-- JOIN specializations sp ON sp.vet_id = v.id
+-- WHERE sp.vet_id IS NULL
+-- GROUP BY v.name;
 -- third
-SELECT vet_name,
-       COUNT(vet_name) AS num_visits
-FROM (
-    SELECT v.name AS vet_name
-    FROM visits vi
-    FULL JOIN vets v ON v.id = vi.vet_id
-    FULL JOIN specializations sp ON sp.vet_id = v.id
-    WHERE sp.vet_id IS NULL
-) AS t
-GROUP BY vet_name;
+-- SELECT vet_name,
+--        COUNT(vet_name) AS num_visits
+-- FROM (
+--     SELECT v.name AS vet_name
+--     FROM visits vi
+--     FULL JOIN vets v ON v.id = vi.vet_id
+--     FULL JOIN specializations sp ON sp.vet_id = v.id
+--     WHERE sp.vet_id IS NULL
+-- ) AS t
+-- GROUP BY vet_name;
```

Comment:
- [ ] Unless the commented code help to the reader we should remove those lines of code that we are not gonna use, remember YAGNI.

Reply:
done